### ISSUE TITLE
No label generated for accessibility

### ISSUE THREAD

**Issue by jongund** [lambdalisue/django-codemirror-widget#25]
We need to have a label generated for the TEXTAREA element that code mirror renders in parallel with the original TEXTAREA element.  The label is very important for supporting people with disabilities and an is required by law for most public organizations to comply with web accessibility standards.

You can use the label text on the original textarea (now hidden by code mirror) defined by a LABEL[FOR] relationship and then use ARIA-LABEL="[label text]" on code mirror TEXTAREA element.
